src/models/multiChart.js

```diff
@@ -571,6 +572,13 @@ nv.models.multiChart = function() {
                 scatters1.interactive(false);
                 scatters2.interactive(false);
             }
+        }},
+
+        transitionDuration: {get: function(){return duration;}, set: function(_) {
```

Comment:
The option should be named the same... just "duration", so that it matches other charts.